### DISCUSSION TITLE
Is there any way to display time in the same line with others?

### DISCUSSION THREAD

**Discussion by LasyIsLazy** [starship/starship#5121] [Q&A] [answered]
In fact, the time shows the time last command finished.
For now, time is in the same line with next command, it looks like the time next command start to execute.

**Comment by mickimnet** [comment, marked as answer]
Take a look at the `[format]` in the [documentation](https://starship.rs/config/#prompt), with that, you can change the order of all modules.

**Reply by LasyIsLazy** [reply]
Thanks for reply, it works.

**Reply by LasyIsLazy** [reply]
There is another thing confusing me: The [Default Prompt Format](https://starship.rs/config/#prompt) shows there is no new line between the time module and other modules. After moving the time module to the first module, it works. I wonder if there is some module ahead of the time module, which will add a new line.

**Reply by mickimnet** [reply]
Check `$line_break\` in the `[format]` section – see: https://starship.rs/config/#line-break
